docs/thirdparty-openapi3-snippets.yaml

```diff
@@ -594,12 +618,44 @@ components:
       description: Scope + Account Identifier mapping for a Consent.
       properties:
         scope:
-          $ref: '#/components/schemas/AuthScopesType'
+          $ref: '#/components/schemas/ConsentScopeType'
         accountId:
           $ref: '#/components/schemas/AccountAddress'
       required:
         - scope
         - accountId
+    ThirdpartyAuthorizationIDPostResponse:
+      title: ThirdpartyAuthorizationIDPostResponse
+      type: object
+      description: 'The response for POST /thirdpartyRequests/transactions/{id}/authorizations request.'
+      properties:
+        challenge:
+          type: string
+          description: The original Challenge Object as a JSON string
```

Comment:
we shouldn't use JSON.stringify as stringify method because it can generate non-identical strings, there is a need for additional design decision see: https://github.com/mojaloop/pisp/issues/66